clvm-traits/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+#[cfg(feature = "derive")]
+pub use clvm_derive::*;
+
+mod error;
+mod from_clvm;
+mod macros;
+mod match_byte;
+mod to_clvm;
+
+pub use error::*;
+pub use from_clvm::*;
+pub use macros::*;
+pub use match_byte::*;
+pub use to_clvm::*;
+
+#[cfg(test)]
+#[cfg(feature = "derive")]
+mod tests {
+    use clvmr::Allocator;
+
+    use super::*;
+
+    #[derive(Debug, ToClvm, FromClvm, PartialEq, Eq)]
+    #[clvm(tuple)]
```

Comment:
I would like these kinds of encodings documented somewhere. Maybe here, along with the tests, is a good place.
i.e. `tuple`, `proper_list` and `curried_args`

Reply:
I'll add it to the crate level docs